docs/_source/getting_started/installation/configurations/elasticsearch.md

```diff
@@ -10,10 +10,7 @@ If you have an Elasticsearch instance and want to share resources with other app
 
 All you need to take into account is:
 
-
-* Argilla will create its ES indices with the following pattern `.argilla*`. It's recommended to create a new role (e.g., argilla) and provide it with all privileges for this index pattern.
-
-* Argilla creates an index template for these indices, so you may provide related template privileges to this ES role.
+* Argilla will create its ES indices with the following patterns: `rg.*` and `*ar.dataset*`. It's recommended to create a new role (e.g., argilla) and provide it with all privileges for these index patterns.
 
 Argilla uses the `ARGILLA_ELASTICSEARCH` environment variable to set the ES connection.
```

Comment:
```suggestion
- Argilla uses the `ARGILLA_ELASTICSEARCH` environment variable to set the ES connection.
```
I would add this as another bullet point in the "things to take into account". Another option is to get rid of the bullet point list, as we only have one item in the list right now.